templates/repo/editor/commit_form.tmpl

```diff
@@ -40,6 +40,7 @@
 				</div>
 			</div>
 			{{if not .Repository.IsEmpty}}
+			{{if not .HideCreatePullRequest}}
```

Comment:
Indent this block.

Reply:
I didn't do that because it will make the PR review more easier.

Reply:
Would still do it, reviewers can hide whitespace, I do that often for template changes.

Reply:
Agree to indent.

If the new code is just one level like:

```
{{if NewCode}}
<div>old content</div>
{{end}}
```

I think it is acceptable.

But now the new code makes it becomes 2 leves:

```
{{if NewCode}}
{{if OldCode}}
<div>old content</div>
{{end}}
{{end}}
```

For this case, IMO indention is a must.